crates/cli/src/main.rs

```diff
@@ -36,9 +36,9 @@ enum Commands {
         #[arg(default_value = "5")]
         bp: String,
 
-        /// Set this flag to use the config.toml.
-        #[arg(short = 'c', long = "config", action)]
-        config: bool,
+        /// Set this flag to use a config.toml.
+        #[arg(short, long, action, default_value = "")]
```

Comment:
it'd be nice to allow the user to just pass the `--config` flag and be able to use the default paths

Reply:
Ahhh will fix.